taca/backup/backup.py

```diff
@@ -286,25 +286,25 @@ def _is_ready_to_archive(self, run, ext):
         if (
             os.path.exists(rta_file)
             and os.path.exists(cp_file)
-            and (not self.file_in_pdc(run.zip_encrypted))
+            and (not self.file_in_pdc(run.tar_encrypted))
         ) or (
             self._get_run_type(run.name) in ["promethion", "minion"]
             and os.path.exists(os.path.join(run_path, ".sync_finished"))
         ):
             # Case for encrypting
-            # Run has NOT been encrypted (run.tar.gz.gpg not exists)
-            if ext == ".tar.gz" and (not os.path.exists(run.zip_encrypted)):
+            # Run has NOT been encrypted (run.tar.gpg not exists)
+            if ext == ".tar" and (not os.path.exists(run.tar_encrypted)):
                 logger.info(
                     f"Sequencing has finished and copying completed for run {os.path.basename(run_path)} and is ready for archiving"
                 )
                 archive_ready = True
             # Case for putting data to PDC
-            # Run has already been encrypted (run.tar.gz.gpg exists)
-            elif ext == ".tar.gz.gpg" and os.path.exists(run.zip_encrypted):
-                logger.info(
-                    f"Sequencing has finished and copying completed for run {os.path.basename(run_path)} and is ready for sending to PDC"
-                )
-                archive_ready = True
+            # Run has already been encrypted (run.tar.gpg exists)
+        elif ext == ".tar.gpg" and os.path.exists(run.tar_encrypted):
```

Comment:
Did you mean to remove one level of indentation here? That changes the logic of the if/elif statements. I'm not sure which one is correct.

Reply:
> Did you mean to remove one level of indentation here? That changes the logic of the if/elif statements. I'm not sure which one is correct.

Good catch! Thanks! It has been fixed